How-To/Create-a-New-Residential-Feature.md

```diff
@@ -0,0 +1,177 @@
+```
+
+-   Once the macro / sql query is correct, run it in your terminal to construct the output in your development database of `AWS Athena`. The output view is the file which will be referenced during `Step 4`. A sample build is presented below.
+
+```         
+dbt build --select proximity.dist_pin_to_stadium
+```
+
+-   Complete the documentation by configuring it as a [source](https://docs.getdbt.com/docs/build/sources) in the folder's dbt `schema.yml` file, and update the folder's `docs.md` and `columns.md` files.
+
+------------------------------------------------------------------------
+
+## Step 4: Incorporate the Transformed Data into the [Final Model View](https://github.com/ccao-data/data-architecture/tree/master/dbt/models/model).
+
+-   Now that the relevant view(s) have been created, they need to be incorporated in the model view.
+
+-   Identify if the new feature is relevant in the condo and/or the residential model. Then, in `dbt/models/model`, update the relevant `vw_pin_condo_input`, `vw_res_card_input`, and `vw_shared_input` view. Within the `model` directory, update the definitions in the appropriate dbt model.
+
+-   If the attribute is spatial, you should also update `crosswalk_year_fill.sql`, `proximity.vw_pin10_proximity_fill.sql`, and `location.vw_pin10_location_fill.sql`. Processes for forward and backward filling are largely conceptual. If we think that the data is time invariant, we can forward fill the data. If we believe that the data has historically been the same, we can backwards fill the data.
+
+------------------------------------------------------------------------
+
+## Step 5: Update the [Model Pipeline](https://github.com/ccao-data/model-res-avm/tree/master) to use the New Versions of the Model Input Views
+
+-   Create a new issue / pull request in the `model-res-avm` repository.
+
+-   Add the new feature to the `predictor` section of `model/models/params.yaml`.
+
+-   If you want to test the impact, you can run the [model ingest steps](https://github.com/ccao-data/model-res-avm?tab=readme-ov-file#usage) with an updated versin of `model.vw_card_res_input`. Make sure that you unfreeze the desired `stage(s)` with `dvc unfreeze {stage}`.
+
+-   To test the impact, you can run the full model with the command
+
+```         
+dvc repro -f
+```
+
+-   If you want this information to be shared, you can push the updated model data to DVC using the following code. If you do this, make sure that the param for run_type is `test`.
+
+```         
+dvc commit
+dvc push 
+```
+
+------------------------------------------------------------------------
+
+## Troubleshooting
+
+-   Some staff types do not have permission to write to AWS Buckets. If your export of data to `aws` returns `Access Denied`, ask senior staff to run the query for you. This is most likely to occur during step 1.
+-   Make sure that you are connected to aws by running `aws-mfa` in the terminal and typing in the correct credentials.
```

Comment:
**[Nitpick, non-blocking]** `aws-mfa` has a purpose that's slightly more specific than "connection":

```suggestion
-   Make sure that you have authenticated to AWS using MFA by running `aws-mfa` in the terminal and typing in the correct credentials.
```